src/payment/ProductLineItems.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+
+import ProductLineItem from './ProductLineItem';
+import { productsSelector } from './data/selectors';
+
+function ProductLineItems({ products }) {
+  return (
+    <React.Fragment>
+      {products.map(product =>
+        (<ProductLineItem
+          product={product}
+          key={product.name}
+        />))}
+    </React.Fragment>
+  );
+}
+
+ProductLineItems.propTypes = {
+  products: PropTypes.arrayOf(PropTypes.shape({
+    imgUrl: PropTypes.string,
+    name: PropTypes.string,
+    seatType: PropTypes.string, // TODO: use PropTypes.oneOf([ all, kinds, of, certs ])
```

Comment:
We know these types now, let's add them here.

Reply:
`PropTypes.oneOf(['professional', 'no-id-professional', 'verified-certificate', 'honor', 'audit'])`